pubspec.yaml

```diff
@@ -2,14 +2,10 @@ name: universal_disk_space
 description: parses total and available disk spaces on Windows and Linux-based systems
 version: 0.1.20
 homepage: https://github.com/joaquimguimaraes/universal_disk_space
-authors:
-  - Gil Nóbrega <jgilfn@gmail.com>
```

Comment:
Removal of authors on a PR is not a good way to impress people 🤣

Reply:
🤷🏻 It is deprecated: https://dart.dev/tools/pub/pubspec#authorauthors

Reply:
This PR is open to maintainers, so the commit can be reverted if it's an issue.

Reply:
Doesn't seem like an issue to me, was just wondering why so. But that clarifies the situation.